usabilla-react-native/src/index.js

```diff
@@ -36,11 +36,20 @@ function setFormDidClose(callback) {
     }
 }
 
+function setCustomVariables(customVariables) {
+    UsabillaBridge.setCustomVariables(customVariables, (errorMessage) => console.log(errorMessage))
```

Comment:
Why do we have an errorMessage callback? it's not handled in the native sdks now. We should only send the dictionary

Reply:
Yes that was a leftover for when we were doing the manual translation from JSON to Map